README.md

```diff
@@ -122,6 +113,19 @@ pip install -r requirements.txt
    - `/ping` : This just answers "hello world" and is a good way to test if your packages are installed correctly without testing Azure itself.
    - `/` : The MSI sample itself
```

Comment:
After setting the environment variables, making sure my service principal has correct permissions, and confirming that a secret named "secret" exists in my vault, this endpoint only brings up an error with the message "vaultBaseUrl". I've tried debugging this for a bit but I'm not sure where the issue is.

Reply:
Resolved -- I had set the vault URL environment variable inside quotations. Removing them made the sample run as expected.